### PR TITLE
homework/lession-3: nguyen van ba

### PR DESCRIPTION
**Pull request by nguyenvanbapc66**

### CODE REVIEW

File: homework/.keep [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (trongtai37) [inline homework/.keep:1]: Should not push redundant files

Comment (nguyenvanbapc66) [reply homework/.keep:1]: i just delete this file

Comment (trongtai37) [reply homework/.keep:1]: Thanks @nguyenvanbapc66, if you have time, pls review my pull request #5

**Comment by quanghoang33** [comment]
it is `lesson` not `lession`

**Pull request merged**